CHANGELOG.md

```diff
@@ -1,4 +1,22 @@
-## 10.1.0 (2017-11-15)
+# 10.2.0 (2017-12-11)
+
+#### :rocket: Enhancement
+* [#376](https://github.com/primer/primer/pull/376) Increase spacing scale. ([@gladwearefriends](https://github.com/gladwearefriends))
```

Comment:
Could you change the title to "Extend spacing scale for marketing" to make it a bit clearer?